src/ert/shared/share/ert/forward-models/res/script/ecl_run.py

```diff
@@ -379,7 +379,12 @@ def runEclipse(self, eclrun_config=None, retries_left=2, backoff_sleep=None):
                 f.write("ECLIPSE simulation complete - NOT checked for errors.")
         else:
             if return_code != 0:
-                raise subprocess.CalledProcessError(return_code, self.sim.executable)
+                command = (
+                    self._get_run_command(eclrun_config)
+                    if self.sim is None
+                    else self._get_legacy_run_command()
```

Comment:
I thought that `_get_legacy_run_command()` should not be used anymore :thinking:

Reply:
Yep, but it has not been removed. It is in use by `flow`.